dashboard/lib/tasks/seed.rake

```diff
@@ -55,6 +55,11 @@ namespace :seed do
     Foorm::Form.setup
   end
 
+  timed_task foorms: :environment do
```

Comment:
seeding a form now directly relies on the library question also being seeded so I made a combined seed step here to ensure they are done in the correct order. I kept the old seed steps in case people want to have a quicker version for testing but can remove those now if desired.

Reply:
Nice idea.  I don't think it hurts to keep the old ones around for a while longer.